PHPgettext/gettext.php

```diff
@@ -98,7 +98,7 @@ function readintarray($count) {
    * @param object Reader the StreamReader object
    * @param boolean enable_cache Enable or disable caching of strings (default on)
    */
-  function __construct($Reader, $enable_cache = true) {
+  function gettext_reader($Reader, $enable_cache = true) {
```

Comment:
@borisson This should be reversed and can you change file permissions back to 664 pls